chapters/CRUD-Operations.md

```diff
@@ -113,20 +124,29 @@ It returns the following output:
 
 #### Resolve Initial DID Document
 
-The Resolve ::Initial DID Document:: algorithm specifies how to resolve an ::Initial
-DID Document:: and validate it against the `identifier` for a specific **did:btc1**.
+This algorithm specifies how to resolve an initial DID document and validate
+it against the `identifier` for a specific **did:btc1**.
 
 It takes the following inputs:
-* `identifier`: A valid **did:btc1** identifier
-* `identifierComponents`: An object containing the following properties
-  * `idType`: the type of identifier (KEY or EXTERNAL)
-  * `version`: the identifier version
-  * `network`: the identifier bitcoin network
-  * `genesisBytes`: the originating public key bytes
-* `resolutionsOptions`: 
+
+* `identifier` - A valid **did:btc1** identifier
+* `identifierComponents` - An object containing the following properties
+  * `idType` - the type of identifier (`KEY` or `EXTERNAL`)
+  * `version` - the identifier version
+  * `network` - the identifier bitcoin network
+  * `genesisBytes` - the originating public key bytes
+* `resolutionsOptions` - optional object that extends the default
+   [DID Resolution Options](https://www.w3.org/TR/did-1.0/#did-resolution-options)
+  * `versionId` - TODO: What exactly is this?
+  * `versionTime` - the time at which the identifier and DID document were created
+  * `sidecarData` - data necessary for resolving a DID such as ::DID Update Payloads:: and ::SMT proofs::.
+  * `network` - the identifier bitcoin network
```

Comment:
This is nicer than above. We should use this style both times I feel.

Also this TODO for versionId. versionId in did:btc1 is an incrementing integer starting from 1, the version of the initial document